multiqc/modules/biscuit/biscuit.py

```diff
@@ -0,0 +1,688 @@
+            dict([(k.replace('_covdist_q40_topgc_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_q40_topgc_table.txt')]),
+            dict([(k.replace('_covdist_cpg_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_cpg_table.txt')]),
+            dict([(k.replace('_covdist_cpg_q40_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_cpg_q40_table.txt')]),
+            dict([(k.replace('_covdist_cpg_q40_botgc_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_cpg_q40_botgc_table.txt')]),
+            dict([(k.replace('_covdist_cpg_q40_topgc_table.txt',''),v) for k, v in self.mdata['coverage'].items() if k.endswith('_covdist_cpg_q40_topgc_table.txt')]),
+        ]
+        if len(mdata)>0:
+            self.add_section(
+                name = 'Cumulative Base Coverage',
+                anchor = 'biscuit-coverage-base',
+                description = "<p>This plot shows the cummulative base coverage. High and low GC content region are the top and bottom 10% 100bp window in GC content.</p>",
+                plot = linegraph.plot(mdata, {'id':'biscuit_coverage_base',
+                    'title': 'Cumulative Base Coverage',
+                    'data_labels': [
+                        {'name': 'All', 'ylab':'Million Bases'}, 
+                        {'name': 'Q40', 'ylab':'Million Bases'}, 
+                        {'name': 'Q40 low GC', 'ylab':'Million Bases'}, 
+                        {'name': 'Q40 high GC', 'ylab':'Million Bases'}, 
+                        {'name': 'CpG (all)', 'ylab':'Million CpGs'}, 
+                        {'name': 'CpG Q40', 'ylab':'Million CpGs'}, 
+                        {'name': 'CpG Q40 low GC', 'ylab':'Million CpGs'}, 
+                        {'name': 'CpG Q40 high GC', 'ylab':'Million CpGs'}, 
+                    ]})
+            )
+            
+            for sid, dd in mdata[0].items():
+                if sid not in basecov:
+                    basecov[sid] = {}
+                basecov[sid]['all'] = dd[1]/dd[0]*100
+
+            for sid, dd in mdata[1].items():
+                if sid not in basecov:
+                    basecov[sid] = {}
+                basecov[sid]['uniq'] = dd[1]/dd[0]*100
+
+            for sid, dd in mdata[4].items():
+                if sid not in basecov:
+                    basecov[sid] = {}
+                basecov[sid]['cpg_all'] = dd[1]/dd[0]*100            
+
+            for sid, dd in mdata[5].items():
+                if sid not in basecov:
+                    basecov[sid] = {}
+                basecov[sid]['cpg_uniq'] = dd[1]/dd[0]*100
+
+        # base coverage >=1x table
+        if len(basecov)>0:
+            self.add_section(
+                name = 'Coverage by At Least One Read',
+                anchor = 'biscuit-coverage-base-table',
+                description = '<p>The fraction covered by at least one read.</p>',
```

Comment:
The fraction of what? No HTML tags needed.

Reply:
`<p>` removed. The description is updated to clarify it's the fraction of either genome or genomic CpGs (depends on tab).